inventory/sample.osp.example.com.d/inventory/group_vars/OSEv3.yml

```diff
@@ -56,6 +56,15 @@ openshift_metrics_cassandra_storage_type: dynamic
 openshift_metrics_cassandra_pvc_size: 40G
 openshift_metrics_duration: 2
 
+# Cinder registry storage
```

Comment:
keep these in `all.yml` as otherwise the post-provision-openstack tools won't find them

Reply:
If we keep these in `all.yml` I believe it conflicts with the implementation in [openshift-ansible-contrib](https://github.com/openshift/openshift-ansible-contrib/blob/master/playbooks/provisioning/openstack/advanced-configuration.md#use-an-existing-cinder-volume-for-the-openshift-registry)  causing the issue described in  #162

Reply:
@day4skiing no, there's no conflict with `contrib` for this. Basically the `post` step doesn't have the necessary values to proceed and hence the individual playbooks cannot be used. Anyway - let's address that in a separate PR to avoid too much cross referencing.